modules/friendlyshared.js

```diff
@@ -75,49 +77,53 @@ Twinkle.shared.callback = function friendlysharedCallback() {
 	Window.display();
 };
 
-Twinkle.shared.standardList = [
-	{
-		label: '{{Shared IP}}: standard shared IP address template',
-		value: 'Shared IP',
-		tooltip: 'IP user talk page template that shows helpful information to IP users and those wishing to warn, block or ban them'
-	},
-	{
-		label: '{{Shared IP edu}}: shared IP address template modified for educational institutions',
-		value: 'Shared IP edu'
-	},
-	{
-		label: '{{Shared IP corp}}: shared IP address template modified for businesses',
-		value: 'Shared IP corp'
-	},
-	{
-		label: '{{Shared IP public}}: shared IP address template modified for public terminals',
-		value: 'Shared IP public'
-	},
-	{
-		label: '{{Shared IP gov}}: shared IP address template modified for government agencies or facilities',
-		value: 'Shared IP gov'
-	},
-	{
-		label: '{{Dynamic IP}}: shared IP address template modified for organizations with dynamic addressing',
-		value: 'Dynamic IP'
-	},
-	{
-		label: '{{Static IP}}: shared IP address template modified for static IP addresses',
-		value: 'Static IP'
-	},
-	{
-		label: '{{ISP}}: shared IP address template modified for ISP organizations (specifically proxies)',
-		value: 'ISP'
-	},
-	{
-		label: '{{Mobile IP}}: shared IP address template modified for mobile phone companies and their customers',
-		value: 'Mobile IP'
-	},
-	{
-		label: '{{Whois}}: template for IP addresses in need of monitoring, but unknown whether static, dynamic or shared',
-		value: 'Whois'
+Twinkle.shared.initialize = function() {
+	if( typeof Twinkle.shared.standardList === 'undefined' ) {
+		Twinkle.shared.standardList = [
```

Comment:
But bear in mind: right now the SharedIP doesn't even sepcify a single message. All messages are defined in the twinklei18n and twinklel10n files.

Is your concern that the l10n file is defining the messages that matter to EN WP and calls them "en" group of messages? And that other English projects may want a different set of options shown? Well, can't they simply modify the l10n file? Wouldn't that be _much easier_ than what they do now, which is the edit 10 different files in 100 different places?